tests/pcmsolver/ccsd-pte/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(pcmsolver-ccsd-pte "psi;quicktests;pcmsolver;addon;ccsd;pte")
```

Comment:
Ok, this branch could use rebasing, the `ccsd` to `cc` could go in then. Otherwise, could get this merged in as-is, then I can fix it in a later PR.